### PR TITLE
feat(grafeas): new proto-only library

### PR DESCRIPTION
**Pull request by coryan**
Grafeas (https://grafeas.io) has a number of "APIs" (mostly protos) that
are used by more than one GCP service.  This change introduces only the
proto library. A future change will start using the library as part of
generated client libraries.

Part of the work for #8106 and #8108

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8111)

### CODE REVIEW

File: google/cloud/grafeas/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+include(GoogleapisConfig)
+set(DOXYGEN_PROJECT_NAME "GrafeasPub/Sub Lite API C++ Client")
+set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for the Pub/Sub Lite API")
```
Comment (dbolduc) [inline google/cloud/grafeas/CMakeLists.txt:19]: pub/sub lite?

File: google/cloud/grafeas/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Grafeas Proto Libraries
+
+:construction:
+
+This directory contains **experimental** libraries. Its contents are subject to
+change without notice.
+
+This directory contains CMake targets to compile the
```
Comment (dbolduc) [inline google/cloud/grafeas/README.md:8]: s/ the// ?

File: google/cloud/grafeas/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+[Grafeas](https://grafeas.io). Grafeas (the Greek word for "Scribe") is
+"An open artifact metadata API to audit and govern your software supply chain".
```
Comment (dbolduc) [inline google/cloud/grafeas/README.md:10]: s/An/an/

also, are the quotes necessary? Consider appending a `¯\(°_o)/¯`

**Review by dbolduc — CHANGES_REQUESTED** [review]

File: google/cloud/grafeas/CMakeLists.txt (hunk 1 above)
Comment (coryan) [reply google/cloud/grafeas/CMakeLists.txt:19]: How else could you know where I cut&pasted this file from?

File: google/cloud/grafeas/CMakeLists.txt [hunk 4]
```diff
@@ -16,7 +16,7 @@
 
 include(GoogleapisConfig)
 set(DOXYGEN_PROJECT_NAME "GrafeasPub/Sub Lite API C++ Client")
```
Comment (dbolduc) [inline google/cloud/grafeas/CMakeLists.txt:18]: also here

**Review by dbolduc — APPROVED** [review]

**Pull request merged**